bin/codemods/src/single-tree-rendering.js

```diff
@@ -15,8 +31,25 @@ export default function transformer( file, api ) {
 		return str.replace( /(import.*\n)\n+(import)/g, '$1$2' );
 	}
 
+	/**
+	 * Ensure `context` is among parameters
+	 */
+	function ensureContextMiddleware( path ) {
+		// `context` param is already in
+		if ( path.value.params.indexOf( 'context' ) !== -1 ) {
```

Comment:
There might be occurrences where it's called `ctx` :slightly_smiling_face:

Reply:
Aside, are there really instances of middleware that don't even have a `context` arg?

Reply:
Hmm, fair enough, there's an example in the PR desc.

Reply:
Searching for missing `context`s was a bit tricky so I simply decided to add this to see if it adds it to anywhere else but this example. :-)

Seems like `ctx` [has been cleaned out already](https://github.com/Automattic/wp-calypso/search?utf8=%E2%9C%93&q=ctx&type=). Good you remembered something like that. :-)